pkgs/by-name/en/encore/go.nix

```
@@ -0,0 +1,87 @@
+{ fetchFromGitHub
+, stdenv
+, lib
+, go
+, ...
+}:
+stdenv.mkDerivation rec {
+  pname = "go-encore";
+  version = "1.22";
+
+  src = fetchFromGitHub {
+    owner = "encoredev";
+    repo = "go";
+    rev = "encore-go${version}";
+    sha256 = "sha256-SSa3CoUS6JMfs6T1PMb3NK8UeJnpLlTIn46/3oCTnL8=";
```

Use of `hash` is preferred over `sha256` as it allows future flexibility should the hash algorithm be changed, e.g. to use `sha512` instead of `sha256`:
```suggestion
    hash = "sha256-SSa3CoUS6JMfs6T1PMb3NK8UeJnpLlTIn46/3oCTnL8=";
```